### ISSUE TITLE
For performance in completeness rate page, Auto switch to Over time if the lowest Administrative area is chosen and disable  location toggle

### ISSUE THREAD

**Issue by SyedaAfrida** [opencrvs/opencrvs-core#3209]
**Bug description:**
For performance in completeness rate page, Auto switch to Over time if the lowest Administrative area is chosen and disable  location toggle

**Steps to reproduce:**
1. Login to any user other than FA
2. Click Performance 
3. Click View of completeness rate
4. Change Location to farajaland 
5. Select locations from toggle
6. Change Location to Ibombo district

**Actual result:**
showing no result

**Expected result:**
Auto switch to Over time if the lowest Administrative area is chosen and disable  location toggle

**Screenshot:**
https://images.zenhubusercontent.com/91778759/e69a86c6-61a4-4f86-947b-9a7d05697634/toggle_location_atdistricts.mp4

**Tested on:**
https://login.farajaland-qa.opencrvs.org/

**Comment by euanmillar** [comment]
@jpye-finch can you let me know if this is still an issue that we need to resolve please?

**Comment by jpye-finch** [comment]
@euanmillar low priority issue.
Design width disabled state for 'By area':
https://www.figma.com/file/HplYu3vGJjAz3u1AmNJ50C/OpenCRVS-Design-Specifications?node-id=6786%3A114764

**Comment by euanmillar** [comment]
Sorry @SyedaAfrida and  @jpye-finch I am still confused about the initial description.  Some users can change location on this page cant they?  Only some users cannot.  The description needs a better explanation for each user type and location type.  What about National Registrars?  What about Performance Managers?

**Comment by jpye-finch** [comment]
there is no restrictions on location search on this view

ticket is just to add disable state at district level (lowest admin level)

**Comment by euanmillar** [comment]
@jpye-finch and @SyedaAfrida offices should also not appear in the completeness rates search

**Comment by SyedaAfrida** [comment]
@euanmillar yes we offices should not appear in completeness rates search as we do not show completeness rate block for offices in the first place

**Comment by SyedaAfrida** [comment]
@anny320 this ticket is fixed and can be closed 

https://user-images.githubusercontent.com/94350336/193253746-2bc2957b-e28a-44f5-afa6-7ceb0ac285bb.mp4

**Comment by anny320** [comment]
completed
